databuilder/extractor/bigquery_usage_extractor.py

```diff
@@ -39,22 +38,23 @@ def init(self, conf):
         self.email_pattern = conf.get_string(BigQueryTableUsageExtractor.EMAIL_PATTERN, None)
         self.delay_time = conf.get_int(BigQueryTableUsageExtractor.DELAY_TIME, 100)
 
-        self.table_usage_counts = {}
+        self.table_usage_counts: Dict[TableColumnUsageTuple, int] = {}
         self._count_usage()
         self.iter = iter(self.table_usage_counts)
 
-    def _count_usage(self):  # noqa: C901
-        # type: () -> None
+    def _count_usage(self) -> None:  # noqa: C901
         count = 0
         for entry in self._retrieve_records():
             count += 1
             if count % self.pagesize == 0:
                 LOGGER.info('Aggregated {} records'.format(count))
 
+            if entry is None:
```

Comment:
do we need this check for mypy?

Reply:
Correct; happy to add a comment as such if it's not clear. Could also cast `entry` as `any` to force an exception if None. I'm new to python types so not sure what's more pythonic :)